### PR TITLE
terraform v0_12 module

### PR DESCRIPTION
**Pull request by bamaralf**

### CODE REVIEW

**Comment by e-moshaya** [comment]
do you have an estimate when this will be merged?

**Comment by antonbabenko** [comment]
Hopefully closer to the end of this week or during weekends.

**Comment by e-moshaya** [comment]
@antonbabenko any update on this?

**Comment by e-moshaya** [comment]
very eager to start using this module, please can I get an update?

**Comment by e-moshaya** [comment]
would also be nice to have an optional create_bucket variable to enable/disable module similar to the create_vpc variable in https://github.com/terraform-aws-modules/terraform-aws-vpc

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                             tags                              = filter.value.tags
+                         }
+                     }
+                     */
+                  }
+                }
+        } 
+    }
+
+    dynamic "server_side_encryption_configuration" {
```
Comment (Ghost---Shadow) [inline main.tf]: Can you do a `terraform fmt` on this file?

**Comment by antonbabenko** [comment]
Thanks @bamaralf and other reviewers!

[v0.0.1](https://github.com/terraform-aws-modules/terraform-aws-s3-bucket/releases/v0.0.1) has been just released.

This module requires some more improvements before it can be `v1.0.0`. I will make those in August when I am back from vacations.

**Comment by bamaralf** [comment]
Hi Ebrahim
Sorry for the late reply
I think that it won't apply to the module as it is on the "AWS vpc" module.
How are you thinking in using it?
Best regards
Bruno Amaral

**Pull request merged**